src/dashboard/Dashboard.js

```diff
@@ -193,9 +195,12 @@ export default class Dashboard extends React.Component {
     }
 
     const AppsIndexPage = () => (
-      <AccountView section='Your Apps'>
-        <AppsIndex newFeaturesInLatestVersion={this.state.newFeaturesInLatestVersion}/>
-      </AccountView>
+      <Translation>
+        {t =>
+          <AccountView section={t('Your Apps')}>
```

Comment:
This looks overly complex, just to translate the string `Your Apps`. Is there a less intrusive way?

Reply:
Yes, but why is the whole translation tag needed, if we are only changing the string

Reply:
https://react.i18next.com/latest/translation-render-prop 
With the trans component， should wrap the <Translation> tag to whole translation tag

Reply:
I have no idea about 'alternative modules'. https://github.com/i18next/react-i18next has 8k+ star. This program has so many 'star', so it should be very popular and have its advantages

Reply:
Looking at the playground code from them, the implementation looks much simpler:

```
const { t, i18n } = useTranslation();
...
return (
   <h2>{t("Welcome to React")}</h2>
)
```

Reply:
ok, got it